makefile

```diff
@@ -188,8 +188,15 @@ endif
 # single-prec codes separate, and not all have one
 EXAMPLES = $(basename $(wildcard examples/*.*))
 examples: $(EXAMPLES)
+ifneq ($(MINGW),ON)
 # this task always runs them (note escaped $ to pass to bash)...
 	for i in $(EXAMPLES); do echo $$i...; ./$$i; done
+else
+# Windows does not find the dynamic libraries, so we make a temporary copy
+	copy $(DYNLIB) examples
```

Comment:
It seems weird that windows can only find a dynlib that is in the same directory. Could there be a compile/link flag that tells it the dynlib (relative?) location, which instead gets added to CFLAGS in the make.inc.windows_mingw ?   Would be much neater than copy & del...

Reply:
I would also prefer temporarily adding .../finufft/lib to PATH; however, I would have to incorporate the build target .ONESHELL. Unfortunately, this target requires a somewhat recent release of make. The one shipping with TDM-GCC 9.2 is too old. For whatever reason, I have not been successful in modifying the environment variable with "set PATH=%PATH%;%cd%\lib & ...". What do you think? Does it make more sense to require a recent version of make or to use my hacky workaround?

Reply:
Ok, we keep your workaround!